### DISCUSSION TITLE
Serverless target creates a very large pages/_error.js

### DISCUSSION THREAD

**Discussion by vbudovski** [vercel/next.js#22822] [Help]
### What version of Next.js are you using?

10.0.8

### What version of Node.js are you using?

12 - 14

### What browser are you using?

N/A

### What operating system are you using?

macOS

### How are you deploying your application?

N/A

### Describe the Bug

When building Next.JS with target `serverless`, the build completes, but produces an `_error` page in the serverless directory that is ~5.7MB in size.

### Expected Behavior

Error page size is more in line with all other generated pages (2k - 50k).

### To Reproduce

Set webpack build target to `serverless` and run `next build`

**Comment by sambecker** [comment]
Not sure if this is related, but moving from `10.0.7` to `10.0.8` made my serverless function too large to deploy to Vercel. Offending function went from `under < 50mb` to `51.9mb`.

**Reply by timneutkens** [reply]
When deploying on Vercel the `serverless` target is not used, so this can't be related. Can you email support@vercel.com so that the issue you're running into can be investigated? Thanks!

**Reply by sambecker** [reply]
Will do!

**Reply by vasco3** [reply]
did they answer?

**Comment by timneutkens** [comment]
`.next/serverless/pages/_error.js` is 996K, same as other pages when using `yarn create next-app`:

```
▲  pages (master) du -sh *
996K	_error.js
996K	index.js
```

It's likely that you're importing some library in `pages/_error.js`. I'll convert this into a help discussion given it's not a bug.

**Reply by vasco3** [reply]
I also see very small sizes when I build it locally. But once I deploy to Vercel it somehow bloats up to 50Mb+. I wonder why. It happens also in `pages/api/` which prevents me from deploying. I also noticed that the `.next/serverless/` files are unminified. That adds bytes

**Comment by vbudovski** [comment]
I have only customised the `_app` and `_document` pages, not `_error`. It turns out that the main cause of the bloat in the generated `_error.js` was actually to do with the liberal use of dynamic importing of components with `ssr: false`. Do you have any insight as to why that happens?

**Comment by vbudovski** [comment]
@timneutkens I've been doing some experimenting and have discovered what I believe to be the source of the large output. I'm using a lot of dynamic imports in my NextJS site which results in a `react-loadable-manifest.json` of ~9.42MB. This then gets inlined inside `serverless/pages/_error.js`. It dwarfs the rest of the contents of that file which weighs in at ~3MB minus the inlined manifest. Any way around that?

**Reply by kholiavko-roman** [reply]
@vbudovski did you found a solution how to reduce `react-loadable-manifest.json` ?
